.github/workflows/build_branch.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           persist-credentials: false
+          fetch-depth: 0
```

Comment:
Denne trengs for å kunne ha mer enn én "canary-publish". Hvis ikke tryner den fordi den allerede finnes. I dag feiler alle bygg etter første commit på en branch.

Reply:
Fint at du fant ut av dette :o